build-tools/pom.xml

```diff
@@ -57,24 +57,6 @@
         <url>https://github.com/awslabs/aws-glue-schema-registry.git</url>
     </scm>
 
-    <build>
```

Comment:
Without the build tools containing jacoco plugin, we were unable to see the coverage report in the past. Are you able to do a clean build and see jacoco coverage report ?

Reply:
not sure why would this affect anything. This is only for executing on this maven module, as there is no code in this module the plugin execution is not doing anything.

I ran clean build and here I can see the report in target/site/jacoco .

e.g. for avro-kafkaconnect-converter module

![image](https://user-images.githubusercontent.com/1935500/224141044-4cf17c36-2e2b-4589-ad77-928076e26f00.png)

Reply:
the plugin configuration is done in parent pom which all of the modules inherit.

Reply:
Would you mind splitting this PR into two, one to just update java versions and one to remove this. I remember that build tools was copying over this configuration to each module since there was no nice way to run coverage on all modules. I will need some time to validate this after cleaning up m2 repository.
Thanks.

Reply:
Ok I'll bring it back for now. This folder doesn't even need to be a maven module at all. it only contains the 2 checkstyle xml files

Reply:
reverted this change